test/NetworkTest.spec.ts

```diff
@@ -207,11 +206,49 @@ describe('Network Test', () => {
 
         const onUpdate = (stats: Stats) => console.info('Subscriber stats:', stats);
 
-        networkTest.testQuality(onUpdate)
+        networkTest.testQuality(null, onUpdate)
           .then(validateResults)
           .catch(validateError)
           .finally(done);
       }, 40000);
+
+      it('should return valid test results or an error when there is no camera', (done) => {
+        const realOTGetDevices = OT.getDevices;
```

Comment:
What about creating another `networkTest` instance with the hacked version of OT instead of doing the reassignment, like we're doing with `badCredentialsNetworkTest`.

Reply:
This would require creating a new version of OT that mangles the `getDevices()` method. And cloning all of OT uses more memory than temporarily swapping out the one method on the original OT object.